src/kdbindings/signal.h

```diff
@@ -242,6 +252,27 @@ class Signal
             return m_connections.insert({ slot });
         }
 
+        // Establish a deferred connection between signal and slot, where ConnectionEvaluator object
+        // used to queue all the connection to evaluate later. The returned
+        // value can be used to disconnect the function again.
```

Comment:
nit:
```suggestion
        // is used to queue all the connection to evaluate later. The returned
        // value can be used to disconnect the slot later.
```

The word `again` doesn't fit here because there's no previous disconnect opration to refer to.